pyscriptjs/src/main.ts

```diff
@@ -187,7 +182,7 @@ export class PyScriptApp {
         this.plugins.afterSetup(interpreter);
 
         //Refresh module cache in case plugins have modified the filesystem
-        interpreter.invalidate_module_path_cache();
+        interpreter._remote.invalidate_module_path_cache();
```

Comment:
my idea is that all the access to `interpreter._remote` should be encapsulated by methods on `InterpreterClient`.
There should be no mention of `._remote` in `main.ts`

Reply:
Yeah, this is the next step @antocuni